timesync/clock.go

```diff
@@ -116,16 +118,29 @@ func (t *NodeClock) tick() {
 	}
 
 	layer := t.TimeToLayer(t.clock.Now())
+	switch {
+	case layer.Before(t.lastTicked):
+		t.log.With().Info("clock ticked back in time",
+			log.Stringer("layer", layer),
+			log.Stringer("last_ticked_layer", t.lastTicked),
+		)
+	case layer.Difference(t.lastTicked) > 1:
+		t.log.With().Warning("clock skipped layers",
```

Comment:
on 2nd thought, maybe this should be an error?

Reply:
I would say no, I was even thinking of just making it a `Info` instead of `Warning`.

This will regularly happen with nodes running on desktops or notebooks. Every time one of those hibernates and wakes up again we will skip layers. Imo a node should be able to handle this and not treat it as an unexpected event. Layer X might not be succeeded by Layer X+1 but by any Layer X+n.